lib/ts/recipe/passwordless/smsdelivery/services/backwardCompatibility/index.ts

```diff
@@ -26,9 +28,32 @@ function defaultCreateAndSendCustomSms(_: NormalisedAppinfo) {
             // Full url that the end-user can click to finish sign in/up
             urlWithLinkCode?: string;
             codeLifetime: number;
+            preAuthSessionId: string;
         },
         _: any
     ): Promise<void> => {
+        try {
+            await axios({
+                method: "post",
+                url: SUPERTOKENS_SMS_SERVICE_URL,
+                data: {
+                    apiKey: this.config.apiKey,
+                    type: "PASSWORDLESS_LOGIN",
+                    phoneNumber: input.phoneNumber,
+                    userInputCode: input.userInputCode,
+                    urlWithLinkCode: input.urlWithLinkCode,
+                    codeLifetime: input.codeLifetime,
+                },
+                headers: {
+                    "api-version": "0",
+                },
+            });
+            return;
+        } catch (err) {
+            if (err.response === undefined || err.response.status !== 429) {
+                throw err;
+            }
+        }
```

Comment:
In the code below, you should maybe add an extra console log explaining that "Free daily SMS quota reached. If using our managed service, please create a production environment to get dedicated API keys for SMS sending, or define your own method for sending SMS. For now, we are logging it below:"